recipe/meta.yaml

```
@@ -41,7 +41,9 @@ requirements:
     - {{ compiler('fortran') }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - make
+    - gnuconfig  # [unix]
+    - make  # [unix]
+    - {{ mpi }}  # [build_platform != target_platform and mpi != 'nompi']
```

Given the error is:

```
ld: symbol(s) not found for architecture x86_64
```

I think you need to add the libraries in `host` to `build` as well, with the same selector (`# [build_platform != target_platform]`) as this line.

So, maybe something like this?

```suggestion
    - {{ mpi }}  # [build_platform != target_platform and mpi != 'nompi']
    - netcdf-fortran  # [build_platform != target_platform]
    - netcdf-fortran * {{ mpi_prefix }}_*  # [build_platform != target_platform]
    - hdf5  # [build_platform != target_platform]
    - hdf5 * {{ mpi_prefix }}_*  # [build_platform != target_platform]
    - libnetcdf  # [build_platform != target_platform]
    - libnetcdf * {{ mpi_prefix }}_*  # [build_platform != target_platform]
```

---

Okay I'll add that.  Thanks. There's no need for the redundant names in `build`, just in `host`, so I'll only add the versions with the appropriate MPI build-string prefix.